charts/reform-scan-blob-router/values.aat.template.yaml

```diff
@@ -3,4 +3,3 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
-  devApplicationInsightsInstrumentKey: "4bb6dd82-2362-4899-acba-56e027dddc49"
```

Comment:
this can stay here ?

Reply:
I don't think so. all resources will be deleted. Recreated in another repository so presuming - different IDs. Same answer for next one: since key vault will be deleted - nothing to load from

Reply:
update the new id once you have created new on ? No relation to key vault this is the app insights key 🤔

Reply:
oh yes